Backend/src/main/java/com/onbank/api/model/csv/TransferToCSV.java

```diff
@@ -0,0 +1,34 @@
+package com.onbank.api.model.csv;
+
+import com.onbank.api.model.Transfer;
+import com.opencsv.bean.ColumnPositionMappingStrategy;
+import com.opencsv.bean.StatefulBeanToCsv;
+import com.opencsv.bean.StatefulBeanToCsvBuilder;
+import com.opencsv.exceptions.CsvDataTypeMismatchException;
+import com.opencsv.exceptions.CsvRequiredFieldEmptyException;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class TransferToCSV {
```

Comment:
Ta klasa rozumiem teraz tworzy csv z podanych obiektów. A gdzie się zapisuje ten plik?

Reply:
W sumie trochę mylącą, bo jeśli podamy tylko nazwę pliku, to powstanie on w głównym katalogu projektu, ale możemy też tam podać katalog.

Reply:
To zróbmy dwa foldery w katalogu resources dla outcoming i incoming tak żeby nie leciało to w cały świat

Reply:
I tam kierujmy csv

Reply:
No dobra, to możemy tak zrobić, ale póki w tych folderach będzie pusto to git nie pozwoli wrzucić ich na repo, więc chyba robimy merge bez tych folderów, tak?

Reply:
A czemu nie pozwoli? Wrzućmy je puste tak żeby były

Reply:
https://git.wiki.kernel.org/index.php/GitFaq#Can_I_add_empty_directories.3F
Git chyba indeksuje tylko pliki z folderów.

Reply:
Na to wygląda, no to na razie zróbmy tylko w kodzie obsługe wrzucania do odpowiedniego folderu pliku, ewentualnie ze stworzeniem go jeśli nie istnieje i tyle. W momencie, gdy pierwszy plik będzie lądował do folderu to nam aplikacja go utworzy